src/main/resources/liquibase/jpcontentscheduler/changeSetPort.xml

```diff
@@ -0,0 +1,21 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+           http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd">
+
+    <changeSet id="00000000000001_jpcontentscheduler_dataPort_test" author="entando" context="test">
+        <sqlFile path="port/00000000000001_dataPort_test.sql" relativeToChangelogFile="true" />
+    </changeSet>
+
+    <changeSet id="00000000000001_jpcontentscheduler_dataPort_production" author="entando" context="production">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">SELECT COUNT(*) FROM databasechangelog WHERE id = '00000000000001_jpcontentscheduler_dataPort_restore'</sqlCheck>
```

Comment:
Use `DATABASECHANGELOG` (uppercase), otherwise this can fail in MySQL

Reply:
Done